managed/services/telemetry/datasource_pmmdb_select.go

```diff
@@ -89,7 +88,7 @@ func openPMMDBConnection(config DSConfigPMMDB) (*sql.DB, error) {
 	db.SetMaxOpenConns(1)
 
 	if err := db.Ping(); err != nil {
-		return nil, fmt.Errorf("connection to PMM DB failed: %s", err)
+		l.Warnf("DB is not reachable [%s]", config.DSN.DB)
```

Comment:
isn't an error appear in line 80?

Reply:
no, on L80 it loads driver for `postgres` and initializes pool.
actual DB availability is checked here.